controllers/rhmi/bootstrapReconciler.go

```diff
@@ -113,6 +113,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	}
 	// MGDAPI-4641 block end
 
+	operatorLimitRangeParams := resources.LimitRangeParams{
+		CpuRequest:    "5m",
+		MemoryRequest: "10Mi",
+		ResourceType:  corev1.LimitTypeContainer,
+	}
```

Comment:
Since these values are used throughout each reconciler and acts as a default, any thoughts on just externalising this to a `var` called `defaultLimitRangeParams` like in https://github.com/carlkyrillos/integreatly-operator/blob/e7ccc0c79e56ab343b6eb27be33727f58ad7fd4e/pkg/resources/limitRange.go#L25 and used as an exported parameter in the reconcile function 🤔